x-pack/plugin/src/test/resources/rest-api-spec/api/ml.post_data.json

```diff
@@ -5,6 +5,10 @@
       "description":"Sends data to an anomaly detection job for analysis."
     },
     "stability":"stable",
+    "headers":{
+      "accept": [ "application/json"],
+      "content_type": ["application/x-ndjson", "application/json"]
```

Comment:
Similar to other APIs that expect newline delimited JSON request body - I think we should be consistent on whether we document `application/json` as accepted Content-Type across these APIs